schemas/config-editor.xsd

```diff
@@ -1154,6 +1155,119 @@ Note: Only sections with forEach support del attribute.
     </xs:unique>
   </xs:element>
 
+
+
+  <xs:element name="list">
+    <xs:annotation>
+      <xs:documentation><![CDATA[
+Adding a list
+-------------
+
+The list can be used to display a list of contact with only organisation name for example.
+This can facilitate building simple form hiding the details of ISO elements. In this case,
+use a contact directory to add new contact makes sense.
+
+This mode can be also more efficient when record contains lots of contacts which in some case
+can create large HTML form which can take time to build, download and render in the browser
+(eg. when having more than 100 contacts in a record). If only `label` are used the form will be faster to render.
+
+eg.
+
+..code-block:: xml
+
+
+        <action type="add"
+                forceLabel="true"
+                addDirective="data-gn-directory-entry-selector"
+                or="pointOfContact"
+                in="/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification">
+          <directiveAttributes
+                  data-template-add-action="false"
+                  data-search-action="true"
+                  data-popup-action="true"
+                  data-template-type="contact"
+                  data-variables="gmd:role/gmd:CI_RoleCode/@codeListValue~{role}"/>
+        </action>
+
+        <list name="Contact point"
+              xpath="/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:pointOfContact[*/gmd:role/*/@codeListValue = 'pointOfContact']"
+              del="."
+              sortBy="gmd:organisationName/gco:CharacterString">
+          <item href="*[1]/@uuid">
+            <label xpath="*/gmd:organisationName/gco:CharacterString/text()"/>
+          </item>
+        </list>
+
+        <list name="Originator"
+              xpath="/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:pointOfContact[*/gmd:role/*/@codeListValue = 'originator']"
+              sortBy="gmd:organisationName/gco:CharacterString">
+          <item>
+            <field xpath="*/gmd:organisationName"/>
+          </item>
+        </list>
```

Comment:
Testing this example, the input box for the organisation, can be removed, but then is removed the organisation, not the contact (in the `label` mode, it is removed the contact, what makes more sense).

It's not something critical, but to confirm this is expected:

![test-organisation-1](https://github.com/geonetwork/core-geonetwork/assets/1695003/f133117b-aa88-468d-9bd6-c4ca0097a018)

![test-organisation-2](https://github.com/geonetwork/core-geonetwork/assets/1695003/0598e7bf-4297-4b63-90d5-09bf580f4b51)

Reply:
Yes, there is no `del` attribute in list, so the item has no button for removal. The org name is a normal field so can be removed as usual. If using 
```
<field xpath="*/gmd:organisationName" del="ancestor::gmd:pointOfContact"/>
```
would remove the contact.

For now, we are more thinking using this mode with directory entries and only label.